dxp/sennajs_integration.md

```diff
@@ -45,17 +45,17 @@ To disable single application mode in your portlet, you can follow these steps:
 
 -   Set the com.liferay.portlet.single-page-application property to false:
 
-    ```properties
-    com.liferay.portlet.single-page-application=false
-    ```
+        ```properties
+        com.liferay.portlet.single-page-application=false
+        ```
 
-    If you prefer, you can set this property to `false` in your `liferay-portlet.xml` instead by adding the following property to the `<portlet>` section:
+        If you prefer, you can set this property to `false` in your `liferay-portlet.xml` instead by adding the following property to the `<portlet>` section:
 
-    ```xml
-    <single-page-application>false</single-page-application>
-    ```
+        ```xml
+        <single-page-application>false</single-page-application>
+        ```
 
-    Alternatively, you can override the `isSinglePageApplication` method of the portlet to return false.
+        Alternatively, you can override the `isSinglePageApplication` method of the portlet to return false.
```

Comment:
I think it was right before:

![Screenshot 2020-10-05 -170739-PAc7RTj5@2x](https://user-images.githubusercontent.com/7074/95097029-4a83b380-072d-11eb-8549-4d6f9119ad3d.png)

This commit makes it look like this:

![Screenshot 2020-10-05 -170704-7sFIeWNz@2x](https://user-images.githubusercontent.com/7074/95096979-3c359780-072d-11eb-9609-9bfa567f357b.png)

(14 commits and counting; we had better squash all this before merging.)

Reply:
I'm going to fix this and squash, I don't know if it's me but each time I try to view the change on GitHub it's messed up